static/cloudformation/kubernetes.template

```diff
@@ -0,0 +1,44 @@
+Parameters:
+  trustedAccount:
+    Type: String
+    Description: The id of the roadie account to allow access.
```

Comment:
Shall we make these more verbose? Where would they find that id?

Reply:
It will be passed in when they click the link from roadie. I can make it more verbose if you have a suggestion?